File: PurchaseOrder.java
public class PurchaseOrder {
    private int regUserFlag;
    private int id;
    private double price;
    private Credit credit;
    private ArrayList<Book> booksReturned;

    // Constructor

    public PurchaseOrder(int regUserFlag, int id, double price, Credit credit, ArrayList<Book> booksReturned) {
        this.regUserFlag = regUserFlag;
        this.id = id;
        this.price = price;
        this.credit = credit;
        this.booksReturned = booksReturned;
    }
    // Getters and Setters

    public int getRegUserFlag() {
        return regUserFlag;
    }

    public void setRegUserFlag(int regUserFlag) {
        this.regUserFlag = regUserFlag;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public Credit getCredit() {
        return credit;
    }

    public void setCredit(Credit credit) {
        this.credit = credit;
    }

    public ArrayList<Book> getBooksReturned() {
        return booksReturned;
    }

    public void setBooksReturned(ArrayList<Book> booksReturned) {
        this.booksReturned = booksReturned;
    }
}

File: model/PurchaseOrder.php
<?php
class PurchaseOrder {
    private $regUserFlag;
    private $id;
    private $price;
    private $address;
    private $booksSold;

    // Constructor

    public function ReturnOrder(&$regUserFlag, &$id, &$price, &$address, &$booksSold) {
        $this->regUserFlag = $regUserFlag;
        $this->id = $id;
        $this->price = $price;
        $this->address = $address;
        $this->booksSold = $booksSold;
    }

    // Getters and Setters

    public function getRegUserFlag() {
        return $this->regUserFlag;
    }

    public function setRegUserFlag(&$regUserFlag) {
        $this->regUserFlag = $regUserFlag;
    }

    public function getId() {
        return $this->id;
    }

    public function setId(&$id) {
        $this->id = $id;
    }

    public function getPrice() {
        return $this->price;
    }

    public function setPrice(&$price) {
        $this->price = $price;
    }

    public function getAddress() {
        return $this->address;
    }

    public function setAddress(&$address) {
        $this->address = $address;
    }

    public function getBooksSold() {
        return $this->booksSold;
    }

    public function setBooksSold(&$booksSold) {
        $this->booksSold = $booksSold;
    }

    public function addBook(&$book){
        array_push($this->booksSold, $book);
    }
}

?>
